### ISSUE TITLE
Behavior of "cf services" and "cf service" is inconsistent with regard to permissions

### ISSUE THREAD

**Issue by neckert** [cloudfoundry/cloud_controller_ng#580]
## Issue

Behavior of "cf services" and "cf service" is inconsistent with regard to permissions

## Context

An organization manager without any space role assigned in the target space can use `cf services` to list all service instances of the space. However, when using `cf service` to show the service instance info for such a service, an error message is displayed that states that the service instance was not found.

## Steps to Reproduce

Assign the organization manager role to a user for an organization. Choose a space (or create one) but don't assign a space role to this user. Create a user-provided service instance in that org / space.

Login with the user above
`cf login`
List all services instances
`cf services`
Show service instance info for the user-provided service
`cf service <name of user-provided service>`

## Expected result

As the user-provided service instance is listed in the output of `cf services`, I would expect that the user can see the service instance info for the user-provided service instance as well.

## Current result

`cf service <name of user-provided service>` returns an error.

## Possible Fix

`cf services` shows only service instances of spaces for which a space role is assigned to the user.

## Example
```
> cf services
Getting services in org Multi Region Team / space dev as nils.eckert@de.ibm.com...
OK

name             service         plan   bound apps   last operation   
logstash-drain   user-provided

> cf service logstash-drain
FAILED
Service instance logstash-drain not found
```

**Comment by simonleung8** [comment]
Hi @neckert 

Thanks for filing the issue. However, we are not able to reproduce the problem using the steps you have provided. User with `OrgManager` role and no Space role can see the service with both commands `cf services` and `cf service`.

We are testing this on `API version 2.55.0` (`cf target` to see version). Can you provide further steps to reproduce the problem?

Thanks
@simonleung8

**Comment by utako** [comment]
Hi @neckert,

Checking back in with you. Can you provide us with more reproduction steps? 

Thanks,
@utako and @simonleung8, CF CAPI Team

**Comment by neckert** [comment]
Hi @utako and @simonleung8 ,

I found the issue on an older version.
I can confirm that the problem is solved on `2.54.0`.

Thanks
@neckert
